app/frontend/entrypoints/application.js

```diff
@@ -1,24 +1,9 @@
 import "../javascript/application"
 
-// To see this message, add the following to the `<head>` section in your
-// views/layouts/application.html.erb
-//
-//    <%= vite_client_tag %>
-//    <%= vite_javascript_tag 'application' %>
-console.log('Vite ⚡️ Rails')
-
-// If using a TypeScript entrypoint file:
-//     <%= vite_typescript_tag 'application' %>
-//
-// If you want to use .jsx or .tsx, add the extension:
-//     <%= vite_javascript_tag 'application.jsx' %>
-
-console.log('Visit the guide for more information: ', 'https://vite-ruby.netlify.app/guide/rails')
-
 // Example: Load Rails libraries in Vite.
 //
-// import * as Turbo from '@hotwired/turbo'
-// Turbo.start()
```

Comment:
Just not sure what to do about this part. The Calendar design looks nice so far though.

Reply:
This application.js is automatically created by Vite when it's installed.
It adds a couple console.log to indicate the javascript is correctly linked when trying the server and it also leaves a bunch of instructions on how to initialize other stuffs.

We don't need those comments not console logs so I just deleted them.
It's not code that would do stuff anyway

Reply:
The red lines just mean I deleted all of that stuff